tensorflow/java/src/gen/gen_ops.bzl

```diff
@@ -0,0 +1,53 @@
+# -*- Python -*-
+
+load("//tensorflow:tensorflow.bzl", "tf_copts")
+
+# Given a list of "ops_libs" (a list of files in the core/ops directory
+# without their .cc extensions), generate Java wrapper code for all operations
+# found in the ops files.
+# Then, combine all those source files into a single archive (.srcjar).
```

Comment:
Exactly. It is not allowed to use wildcards (e.g. `ops/array/*.java`) in the `outs` argument of a `genrule` and since we don't know at the Bazel level which files are to be generated, we cannot list them statically neither.

Also, reading at the Java rules documentation site, it seems to be the right way doing it: 

> Source files of type .srcjar are unpacked and compiled. (This is useful if you need to generate a set of .java files with a genrule.)